gcs/src/main/java/com/google/cloud/hadoop/fs/gcs/GoogleHadoopFileSystemBase.java

```diff
@@ -1856,7 +1865,7 @@ public Map<String, byte[]> getXAttrs(Path path) throws IOException {
   public Map<String, byte[]> getXAttrs(Path path, List<String> names) throws IOException {
     return GhfsStorageStatistics.trackDuration(
         storageStatistics,
-        GhfsStatistic.INVOCATION_XATTR_GET_MAP,
+        GhfsStatistic.INVOCATION_XATTR_GET_NAMED_MAP,
```

Comment:
why this change?

Reply:
The earlier implementation was wrong.  it should have been NAMED_MAP.